src/main/java/org/spongepowered/common/registry/util/RegistryModuleLoader.java

```
@@ -97,6 +100,21 @@ private static boolean hasCatalogRegistration(RegistryModule module) {
                 return true;
             }
         }
+        if (module instanceof AlternateCatalogRegistryModule) {
+            try {
+                final Type type = module.getClass().getGenericSuperclass();
+                Class<?> catalogClass = (Class<?>) ((ParameterizedType) type).getActualTypeArguments()[0];
+                CatalogedBy catalogedBy = catalogClass.getAnnotation(CatalogedBy.class);
+                if (catalogedBy != null) {
+                    Class<?>[] catalogedBys = catalogedBy.value();
+                    if (catalogedBys.length > 0) {
+                        return true;
+                    }
+                }
+            } catch (Exception e) {
+                // Something went wrong
```

Don't just silently ignore it. Print it out to the sponge error log.